ankihub/ankihub_client/ankihub_client.py

```diff
@@ -1290,6 +1290,14 @@ def send_daily_card_review_summaries(
         if response.status_code != 201:
             raise AnkiHubHTTPError(response)
 
+    def is_premium_user(self) -> bool:
```

Comment:
Could you add the `is_trialing` info, because the chatbot is allowed for users who are also on trial period.